pkg/webui/components/field/index.js

```diff
@@ -263,6 +258,9 @@ Field.propTypes = {
    * This is necessary to map form values correctly.
    */
   form: PropTypes.bool,
+  /** The value name that the field will set to touched (defaults to 'name' prop)
+   */
+  touches: PropTypes.string,
```

Comment:
```suggestion
  touched: PropTypes.string,
```

Reply:
This is actually supposed to be called `touches`. There were other PropType definitions missing though (including `touched`) with the new changes, which I've added.

Reply:
So, what is the purpose of the `touches` prop? Could you give an example?

Reply:
Consider the mapping between the `<Field>`'s `name` prop and the form values stored by formik. Usually, we want the corresponding value to be touched when the field is being touched. In case of the `<FieldGroup />`, it usually refers to a single value of the form (like a `rights` array, or later also a enum values set by radio buttons), while the children of the group have a name concatenated from the group name and the name of the field. However, if these fields are touched, we want the whole group to be considered touched.
So in these cases, the FieldGroup will pass the `touches` prop with its own `name` to the Field, to make sure the FieldGroup's value is being set to touched, instead of the value associated to the field itself.

This is also assuming that we will be using FieldGroups for checkboxes or radio-buttons only, although it could be extended to distinguish later on. Setting touch flags for a single checkbox or radio button does not really make sense.